internal/graph/checkingresourcestream.go

```diff
@@ -455,12 +456,13 @@ func (crs *checkingResourceStream) runProcess(alwaysProcess bool) (bool, error)
 		crs.ctx,
 		crs.checker,
 		computed.CheckParameters{
-			ResourceType:  crs.req.ObjectRelation,
-			Subject:       crs.req.Subject,
-			CaveatContext: crs.req.Context.AsMap(),
-			AtRevision:    crs.req.Revision,
-			MaximumDepth:  crs.req.Metadata.DepthRemaining,
-			DebugOption:   computed.NoDebugging,
+			ResourceType:         crs.req.ObjectRelation,
+			Subject:              crs.req.Subject,
+			CaveatContext:        crs.req.Context.AsMap(),
+			AtRevision:           crs.req.Revision,
+			MaximumDepth:         crs.req.Metadata.DepthRemaining,
+			DebugOption:          computed.NoDebugging,
+			TraversalBloomFilter: singleflight.MustNewTraversalBloomFilter(uint(crs.req.Metadata.DepthRemaining)),
```

Comment:
Don't use a `Must` call here; make it return a spicerrors.MustBugf and then return it here so it doesn't panic in prod